.github/release.yml

```diff
@@ -11,14 +11,17 @@ changelog:
         - major
     - title: New Features & Bugfixes 🎉
       labels:
-        - defects
+        - bug
         - enhancements
-        - feature-request
-    - title: Docs & Maintenance 🔧
+        - feature request
```

Comment:
The label has a white space, not sure if the `-` replaces that in yml or not?

Reply:
The space is probably the right way to do it